src/Core/Geographic/CRS.js

```diff
@@ -0,0 +1,122 @@
+import proj4 from 'proj4';
+
+proj4.defs('EPSG:4978', '+proj=geocent +datum=WGS84 +units=m +no_defs');
+
+const UNIT = {
+    DEGREE: 1,
+    METER: 2,
+};
+
+function is4326(crs) {
+    return crs.indexOf('EPSG:4326') == 0;
+}
+
+function _unitFromProj4Unit(projunit) {
+    if (projunit === 'degrees') {
+        return UNIT.DEGREE;
+    } else if (projunit === 'm') {
+        return UNIT.METER;
+    } else {
+        return undefined;
+    }
+}
+
+function toUnit(crs) {
+    switch (crs) {
+        case 'EPSG:4326' : return UNIT.DEGREE;
+        case 'EPSG:4978' : return UNIT.METER;
+        default: {
+            const p = proj4.defs(crs);
+            if (!p) {
+                return undefined;
+            }
+            return _unitFromProj4Unit(p.units);
+        }
+    }
+}
+
+function toUnitWithError(crs) {
+    const u = toUnit(crs);
+    if (crs === undefined || u === undefined) {
+        throw new Error(`Invalid crs parameter value '${crs}'`);
+    }
+    return u;
+}
+
+/**
+ * This module provides basic methods to manipulate a CRS (as a string).
+ *
+ * @module CRS
+ */
+export default {
+    /**
+     * Units that can be used for a CRS.
+     *
+     * @enum {number}
+     */
+    UNIT,
+
+    /**
+     * Assert that the CRS is valid one.
+     *
+     * @param {string} crs - The CRS to validate.
+     *
+     * @throws {Error} if the CRS is not valid.
+     */
+    isValid(crs) {
+        toUnitWithError(crs);
+    },
+
+    /**
+     * Assert that the CRS is geographic.
+     *
+     * @param {string} crs - The CRS to validate.
+     * @return {boolean}
+     * @throws {Error} if the CRS is not valid.
+     */
+    isGeographic(crs) {
+        return (toUnitWithError(crs) != UNIT.METER);
+    },
+
+    /**
+     * Assert that the CRS is geocentric.
+     *
+     * @param {string} crs - The CRS to validate.
+     * @return {boolean}
+     * @throws {Error} if the CRS is not valid.
+     */
+    isGeocentric(crs) {
```

Comment:
Geocentric doesn't mean that the unit is metric. Where is used this function?
By example 'EPSG:4978' is geocentric, but 'EPSG:3857' isn't geocentric while they are in metric unit.

Reply:
It is used in `Coordinates`, `OBB` and `Ex